Fortinet/fortigate/ingest/parser.yml

```
@@ -89,6 +89,7 @@ stages:
           fortinet.event.desc: "{{parsed_event.message.desc}}"
           http.request.method: "{{parsed_event.message.method}}"
           log.level: "{{parsed_event.message.level or parsed_event.message.FTNTFGTlevel or parsed_event.message.FortinetFortiGatelevel}}"
+          log.syslog.hostname: "{{parsed_event.devname}}"
```

I think the proper ECS fields is `observer.hostname`. `log.syslog.hostname` contains the hostname extracted from the header of a syslog message.
```suggestion
          observer.hostname: "{{parsed_event.devname}}"
```